### ISSUE TITLE
Dynamic help text for Mass slider controls is backward

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#80]
The help text is correct in Gravity Force Lab, but in Gravity Force Lab Basics, the strings are the wrong way around.

The help text should say "change density" when Constant Size is checked and "change mass" when Constant size is unchecked.

Likely just a little copy and paste error. Please re-arrange like it is in GFL.
<img width="861" alt="screen shot 2018-12-12 at 10 08 43 pm" src="https://user-images.githubusercontent.com/1902116/49899069-f1ada180-fe5a-11e8-89eb-c1aa574702d9.png">
<img width="862" alt="screen shot 2018-12-12 at 10 10 08 pm" src="https://user-images.githubusercontent.com/1902116/49899075-f5412880-fe5a-11e8-9bfa-4859d13cb0fc.png">

**Comment by terracoda** [comment]
@mbarlow12, I should have referenced the issue over in GFL. Here is the related issue
https://github.com/phetsims/gravity-force-lab/issues/123

**Comment by mbarlow12** [comment]
@zepumph I agree this is an easy switch. I used a lot of ternary ops in the sims, so I think this will be a very quick fix.

**Comment by zepumph** [comment]
simple swap, please review @terracoda.

**Comment by terracoda** [comment]
The swap is perfect. Thanks @mbarlow12 and @zepumph!
Closing.
